q2_feature_table/_filter.py

```diff
@@ -89,11 +89,22 @@ def filter_features(table: biom.Table, min_frequency: int=0,
     return table
 
 
-def filter_seqs(data: pd.Series, metadata: qiime2.Metadata, where: str=None,
+def filter_seqs(data: pd.Series, metadata: qiime2.Metadata=None,
```

Comment:
Are you good with this signature order, or do you think we should stick the two inputs together? Here is how `q2cli` renders it right now:

![screen shot 2017-12-20 at 10 27 57 am](https://user-images.githubusercontent.com/274668/34220353-97d0e03a-e571-11e7-8d23-7c6b290629b7.png)